graphql/aggregator.js

```diff
@@ -72,11 +72,16 @@ export default class Aggregator {
 
     try {
       this.nearVectorString = new NearVector(nearVectorObj).toString();
+      this.includesNearMediaFilter = true;
     } catch (e) {
       this.errors = [...this.errors, e];
     }
 
-    this.includesNearMediaFilter = true
+    return this;
+  };
+
+  withObjectLimit = (objectLimit) => {
+    this.objectLimit = objectLimit;
```

Comment:
maybe this should be somehow verified to be number? or even positive number

Reply:
good point, just pushed validation for this method